ui/guest/src/store/epics/root.ts

```diff
@@ -849,6 +849,13 @@ const moveComponent = (dragContext) => {
           containerRecord.fieldId.includes('.') ? `${containerRecord.index}.${targetIndex}` : targetIndex
         );
       });
+    } else {
+      // if draggedElementIndex equals targetIndex it means that item wasn't moved, but it was locked, so it needs
+      // to be unlocked.
+      const models = contentController.getCachedModels();
+      const id = dragged.modelId;
+      const path = models[id].craftercms.path;
+      post(unlockItem({ path }));
```

Comment:
Check this for embedded components too. You may need to re-retrieve the "parent id" to get to the path in such cases.